lte/gateway/python/magma/mobilityd/rpc_servicer.py

```diff
@@ -182,25 +181,38 @@ def AllocateIPAddress(self, request, context):
                 context.set_details('No free IPv4 IP available')
                 context.set_code(grpc.StatusCode.RESOURCE_EXHAUSTED)
             except DuplicatedIPAllocationError:
-                context.set_details('IP has been allocated for this subscriber')
+                context.set_details(
+                    'IP has been allocated for this subscriber')
                 context.set_code(grpc.StatusCode.ALREADY_EXISTS)
             except DuplicateIPAssignmentError:
-                context.set_details('IP has been allocated for other subscriber')
+                context.set_details(
+                    'IP has been allocated for other subscriber')
                 context.set_code(grpc.StatusCode.ALREADY_EXISTS)
-        else:
-            self._unimplemented_ip_version_error(context)
+        elif request.version == AllocateIPRequest.IPV6:
+            try:
+                ip = self._ip_address_man.alloc_ipv6_address(composite_sid)
+                logging.info("Allocated IPv6 %s for sid %s for apn %s"
+                             % (ip, SIDUtils.to_str(request.sid), request.apn))
+                ip_addr.version = IPAddress.IPV6
+                ip_addr.address = ip.packed
+            except MaxCalculationError:
+                context.set_details('Could not calculate IID for IPv6 address')
+                context.set_code(grpc.StatusCode.RESOURCE_EXHAUSTED)
+            return AllocateIPAddressResponse(ip_addr=ip_addr, vlan="")
         return AllocateIPAddressResponse()
 
     @return_void
     def ReleaseIPAddress(self, request, context):
```

Comment:
Same here, Add support to release both IP address.

Reply:
I'd prefer to maintain this as of now (just releasing one IP address per call) I can add it as TODO and update this on a following PR